src/components/Breadcrumbs/utils.ts

```diff
@@ -0,0 +1,13 @@
+import { Breadcrumb } from '../../hooks/useCioBreadcrumb';
+
+// eslint-disable-next-line import/prefer-default-export
+export const splitBreadcrumbs = (breadcrumbs: Breadcrumb[] = []) => {
+  const [firstElement, secondElement, ...rest] = breadcrumbs;
+  const lastItems = rest.slice(-2);
+  const middle = rest.slice(0, rest.length - 2);
+  return {
+    firstItems: [firstElement, secondElement].filter(Boolean),
```

Comment:
Yeah, so the breadcrumbs are being generated on the `useCioBreadcrumbs` function based on the parent groups array, so we would never get `null/undefined` breadcrumbs _in the middle_ of the array, the `filter(Boolean)` is here to handle scenarios with 0 or 1 breadcrumbs:

1. If we pass in an array that only has 1 element (or zero, for that matter) we would get `[firstBreadcrumb, undefined]` after destructuring, so the filter would get rid of the second element and `rest` would be an empty array. Since `rest` is empty, `middle` will be empty and (`rest.slice(0, rest.length - 2);` is empty) and `lastItems` is also empty (`rest.slice(-2);`)

2. if we pass an array with three or four breadcrumbs
`const [firstElement, secondElement, ...rest] = breadcrumbs;` takes the first two and `rest` is left with exactly two items.
`const lastItems = rest.slice(-2);` takes the last (and only) two items in the array
and `const middle = rest.slice(0, rest.length - 2);` results on an empty array

3. If we pass more than 4, it's going to do the same as above, but `const middle = rest.slice(0, rest.length - 2);` will have a valid range because `rest.length` will be higher than 2.

Reply:
Ah yea makes sense for the 1 breadcrumb scenario! LGTM